### PR TITLE
test(sbb-file-selector): add visual tests

### PR DESCRIPTION
**Pull request by DavideMininni-Fincons**

### CODE REVIEW

File: src/elements/file-selector/file-selector.visual.spec.ts [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+import {
+  describeEach,
+  describeViewports,
+  visualDiffDefault,
+  visualRegressionFixture,
+} from '../core/testing/private.js';
+
+import '../form-error.js';
+import './file-selector.js';
+import type { SbbFileSelectorElement } from '@sbb-esta/lyne-elements/file-selector/file-selector';
```
Comment (TomMenga) [inline src/elements/file-selector/file-selector.visual.spec.ts:12]:
```suggestion
import type { SbbFileSelectorElement } from './file-selector.js';
```

File: src/elements/file-selector/file-selector.visual.spec.ts [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+  };
+
+  describeViewports({ viewports: ['small', 'medium'] }, () => {
+    describeEach(states, ({ variant, state }) => {
+      beforeEach(async function () {
+        root = await visualRegressionFixture(html`
+          <sbb-file-selector
+            id="fs"
+            title-content="Title"
+            ?multiple=${true}
```
Comment (TomMenga) [inline src/elements/file-selector/file-selector.visual.spec.ts:53]:
```suggestion
            multiple
```

File: src/elements/file-selector/file-selector.visual.spec.ts [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+      { disabled: false, error: true },
+    ],
+  };
+
+  const sizes = {
+    variant: ['default', 'dropzone'],
+    size: ['s', 'm'],
+  };
+
+  describeViewports({ viewports: ['small', 'medium'] }, () => {
```
Comment (TomMenga) [inline src/elements/file-selector/file-selector.visual.spec.ts:46]:
```suggestion
  describeViewports({ viewports: ['zero', 'medium'] }, () => {
```
Usually, we use zero/medium

File: src/elements/file-selector/file-selector.visual.spec.ts [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+      );
+    });
+
+    describeEach(sizes, ({ variant, size }) => {
+      beforeEach(async function () {
+        root = await visualRegressionFixture(html`
+          <sbb-file-selector
+            id="fs"
+            title-content="Title"
+            ?multiple=${true}
```
Comment (TomMenga) [inline src/elements/file-selector/file-selector.visual.spec.ts:80]:
```suggestion
            multiple
```

File: src/elements/file-selector/file-selector.visual.spec.ts [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+            ?disabled=${state.disabled}
+          ></sbb-file-selector>
+          ${state.error
+            ? html`<sbb-form-error slot="error">There has been an error.</sbb-form-error>`
+            : nothing}
+        `);
+      });
+
+      it(
+        visualDiffDefault.name,
```
Comment (TomMenga) [inline src/elements/file-selector/file-selector.visual.spec.ts:64]: Should we also test the focus state?

**Review by TomMenga — CHANGES_REQUESTED** [review]
Gg
Some minor fixes

File: src/elements/file-selector/file-selector.visual.spec.ts (hunk 3 above)
Comment (DavideMininni-Fincons) [reply src/elements/file-selector/file-selector.visual.spec.ts:46]: Yes, but in zero the file's icons are cropped, so i went with small
https://lyne-storybook.app.sbb.ch/?path=/story/elements-sbb-file-selector--default&globals=backgrounds.grid:!false;viewport:zero

File: src/elements/file-selector/file-selector.visual.spec.ts [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+        }),
+      );
+
+      it(
+        visualDiffFocus.name,
+        visualDiffFocus.with((setup) => {
+          addFilesToComponentInput(root.querySelector('#fs')!);
+          setup.withSnapshotElement(root);
+        }),
+      );
```
Comment (TomMenga) [inline src/elements/file-selector/file-selector.visual.spec.ts:102]: Nitpick
You can use a for (just to be consistent with other files)
```
for diff of [visualDiffDefault, visualDiffFocus] {
  ...
}
```

**Review by TomMenga — APPROVED** [review]
LGMT 👍 
Just a nitpick

**Pull request merged**